anago

```diff
@@ -742,6 +743,23 @@ get_build_candidate () {
                                 $PARENT_BRANCH \
    || return 1
 
+  # When creating alphas, make sure that X.X.0 has been created first or warn.
+  # This is to ensure that branchff can continue to be used as needed.  User can
+  # override here.
+  [[ $JENKINS_BUILD_VERSION =~ ${VER_REGEX[release]} ]]
+  latest_official=${BASH_REMATCH[1]}.$((${BASH_REMATCH[2]}-1))
+  # The ^{tag} construct looks specifically for a tag (vs. any object)
+  if ! git rev-parse "$latest_official.0^{tag}" >/dev/null 2>&1; then
```

Comment:
will this logic throw a warning when alpha.0 is created (when we create a release branch)?